app/models/profile.rb

```diff
@@ -2,7 +2,7 @@ class Profile < ApplicationRecord
   PROFILE_PERMIT = [:position, :address, :email, :phone, :education, :avatar,
                     qualifications_attributes:[:title, :content, :start_time, :end_time, :_destroy], 
                     experiences_attributes:[:title, :description, :_destroy],
-                    profile_skills:[:name, :point, :_destroy]].freeze
+                    profile_skills_attributes:[:name, :point, :_destroy]].freeze
```

Comment:
```suggestion
                    profile_skills_attributes: [:name, :point, :_destroy]].freeze
```
sau dấu `:` để 1 space